elections/steering/2020/divya-mohan.md

```diff
@@ -16,7 +16,7 @@
 Most of my work within the Kubernetes org relates to pruning and editing - whether it be processes or their underlying documentation.
 
 - Dedicated contributor to k/website repository
-- k8s storyteller on the upstream marketing team - https://github.com/kubernetes/community/tree/master/communication/marketing-team
+- k8s storyteller on the upstream marketing team - https://github.com/kubernetes/community/tree/master/communication/contributor-comms
```

Comment:
This is Divya's personal profile for the 2020 steering election. We should leave it alone for our records.

Reply:
done